src/main/kotlin/uk/gov/gdx/datashare/enums/EnrichmentField.kt

```diff
@@ -3,6 +3,8 @@ package uk.gov.gdx.datashare.enums
 import com.fasterxml.jackson.annotation.JsonValue
 
 enum class EnrichmentField(@JsonValue val jsonName: String) {
+  SOURCE_ID("sourceId"),
```

Comment:
That's an interesting point. Do we consider death source ID to be the same type as prison source ID?

Reply:
yes as its the same in the POST event (supplier) payload